modules/node_modules/@colony/purser-core/helpers.js

```diff
@@ -413,11 +414,59 @@ export const messageOrDataValidator = (
     : messageData;
 };
 
+/**
+ * In order to support EIP-155, it's necessary to specify various
+ * definitions for a given chain (e.g. the chain ID, network ID, hardforks).
+ *
+ * Given a chain ID, this function returns a chain definition in the format
+ * expected by `ethereumjs-tx`.
+ *
+ * @param {number} chainId The given chain ID (as defined in EIP-155)
+ * @return {Object} The common chain definition
+ */
+export const getChainDefinition = (chainId: number): {| common: Object |} => {
+  const baseChain = (() => {
+    switch (chainId) {
+      /*
+       * Ganache's default chain ID is 1337, and is also the standard for
+       * private chains. The assumption is taken here that this inherits
+       * all of the other properties from mainnet, but that might not be
+       * the case.
+       *
+       * @TODO Provide a means to specify all chain properties for transactions
+       */
+      case 1:
+      case 1337:
```

Comment:
These case values should have been taken from the core defaults: 

https://github.com/JoinColony/purser/blob/15721bcff5de6d76e75e092290e1e5ec5cb96008/modules/node_modules/%40colony/purser-core/defaults.js#L125